static/app/components/performance/searchBar.tsx

```diff
@@ -193,7 +193,9 @@ function SearchBar(props: SearchBarProps) {
   const handleSearch = (query: string, asRawText: boolean) => {
     setSearchResults([]);
     setSearchString(query);
-    const fullQuery = asRawText ? query : `transaction:${query}`;
+    query = new MutableSearch(query).formatString();
```

Comment:
you might be missing a bracket here
```suggestion
    query = (new MutableSearch(query)).formatString();
```

Reply:
The formatter removes the brackets if I do that, pretty sure it's still valid syntax as is!

Reply:
really? oh okay then haha